src/basecamp.cpp

```diff
@@ -23,24 +68,98 @@ std::string basecamp::board_name() const
     return string_format( _( "%s Board" ), name.c_str() );
 }
 
-std::string basecamp::save_data() const
+#include <stdio.h>
```

Comment:
Shouldn't that be at the top of the file?

Reply:
temporary debugging code.